site/docs/pages/config/is-base.mdx

```diff
@@ -30,4 +30,4 @@ true;
 
 ## Parameters
 
-[`isBaseOptions`](/config/types#isbaseoptions)
+[`IsBaseOptions`](/config/types#isbaseoptions)
```

Comment:
The site docs now live in the [`@base/web` repo](https://github.com/base/web/tree/master/apps/base-docs/docs/pages/builderkits/onchainkit).